### DISCUSSION TITLE
[Feature request] iOS share target

### DISCUSSION THREAD

**Discussion by igorkulman** [standardnotes/forum#2573] [Suggestions]
Would be nice to work as a share target in iOS for text and links. 

The scenario is as follows

- User select some text in any iOS app and taps Share
- User selects the app as the share target
- The app asks the user to select an existing note or create a new one with a title
- User selects a note
- The apps adds the text at the end of the selected note

This should work the same with links from Safari or any browser, ideally adding the ability to edit the link so the user can write some additional text to the link.

This comes from my own workflow, when I find some link on Twitter, RSS, etc and I want to save it for checking it out later a a "Check out" note in the app with some comment.

**Comment by atmoio** [comment]
Yup, mobile deeplinking is on the radar.

**Comment by rcadariu** [comment]
plus one on this one, similar to what folks describe here https://www.howtogeek.com/232489/how-to-customize-the-ios-sharing-menu/, I see standardnotes ain't making an appearance on my menu

**Comment by JaspalSuri** [comment]
Hi @rcadariu, this is still on our roadmap for features that we would like to add to the apps before reaching version 4. 🙂 That said, it could take some time to implement since it might be tricky adding content into an app that encrypting content locally.
